src/predicates/propEq.js

```diff
@@ -3,14 +3,27 @@
 
 const curry = require('../core/curry')
 const equals = require('../core/equals')
-const isObject = require('../core/isObject')
+const isDefined = require('../core/isDefined')
+const isEmpty = require('../core/isEmpty')
+const isInteger = require('../core/isInteger')
+const isNil = require('../core/isNil')
+const isString = require('../core/isString')
 
-// propEq: (String | Number) -> a -> Object -> Boolean
+// propEq: (String | Integer) -> a -> b -> Boolean
```

Comment:
Apologies, totally slipped my mind how this could be used to compare other properties too :-)

Reply:
No big deal at all.  The error was with `hasProp` and it was missed in review as well. There are a lot of moving parts to this lib so things like this happen. I obviously do it all the time as in the case with `hasProp`.